### ISSUE TITLE
State & Responsive Descriptions for My Challenge number spinners

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#195]
In today's design meeting we discussed new labels for the number spinners:
- Set Left Value
- Set Right Value

AND a new pattern for the Object Response that explicitly states the ratio:
- {{number-spinner-value}} challenge ratio, {{left-value}} to {{right-value}}
    - 1, challenge ratio 1 to 2
    - 2, challenge ratio 1 to 2

AND a new pattern for Context Responses as the new values are set:
- Hands, {{proximity-to-ratio}} challenge ratio. 

In addition, we decided to build an indication of min and max values into the aria-valuetext:
- 1, at min, ratio challenge 1 to {{2}}
- 10, at max, ratio challenge {{5}} to 10

More examples are in the table in the design document here:
[Object & Context Responses - Setting Left & Right Values under My Challenge (screen 2)](https://docs.google.com/document/d/1HJvvGWYaUv9tlat9S7omQQVng8fGB4FuUhTEM1qx1z8/edit#heading=h.ec8j83juule0)

**Comment by zepumph** [comment]
All 4 things above have been implemented, see phettest or master for review.

**Comment by zepumph** [comment]
Please note that this:

>Hands, {{proximity-to-ratio}} challenge ratio.


Now applies to the context response on the first screen when the combo box changes the target also.

**Comment by terracoda** [comment]
Ok, I'll have a listen.

**Comment by terracoda** [comment]
@zepumph, @Matthew-Moore240 and I agreed on iterations for the Ratio Challenge combobox over in #196, and here I'll outline our agreed upon changes for the My Challenge number spinners.

Changes were implemented and here’s two example initial object responses from VO. Of course, all AT is different, but this is what VO says:
![Screen Shot 2020-10-16 at 12 23 54 PM](https://user-images.githubusercontent.com/1902116/96298474-10d46780-0fcd-11eb-91d8-4d6015caee42.png)
![Screen Shot 2020-10-16 at 2 23 09 PM](https://user-images.githubusercontent.com/1902116/96298484-15008500-0fcd-11eb-98cf-0dc0b3434c2d.png)
On initial focus the region is announced which makes initial focus quite wordy.
Note, there was a missing comma after “min”, so the text ran together a bit.

**After listening I made these Testing Notes, updated the design doc, and spoke to Matthew**
I think these items are too much for one issue, so I will create new issues where noted.

1. Could add a common after “min”, but decided to drop the “at min” and “at max” and leave that to sonification. Initially, I didn’t realize there were sounds for that.
2. Remove “Set” from label and use it the help text instead: 
    - Change labels to “Left Value” and “Right Value”
    - Change help text to: Set left and right values to create your challenge, then move hands to explore it.
3. Create responses for the accordion expand/collapse interaction.
    - Communicate the current challenge when box is actively expanded
        - “Current challenge ratio {{1}} to {{2}}.
    - Communicate that the current challenge is hidden when box is actively collapsed
        - “Current challenge hidden.”
    - [ ] TS to create new issue for this item
3. Why don’t we call it “My Challenge Ratio” in the visual sim? 
    - Do you we need to use “My Challenge Ratio” in the non-visual experience? I prefer they are the same. 
    - [ ] TS to create design issue for this tweak.
4. Technical changes to “My Challenge”?
    - There must be some “grouping” logic that is causing the “My Challenge Region” to be spoken on first entering the My Challenge box. I am wondering if it’s critical, or maybe we can remove it?
    - [ ] TS to create separate issue for this.

@zepumph, I updated all the tables in the tables [My Challenge (screen 2)](https://docs.google.com/document/d/1HJvvGWYaUv9tlat9S7omQQVng8fGB4FuUhTEM1qx1z8/edit#heading=h.ec8j83juule0) section that provide changes for this issue.

@zepumph, I think items 1,2, and 3 all pertain to this issue, so I will summarize the changes and you can double check the updated tables if you need to.

- [ ] Change the labels of the number spinners to
- Left Value
- Right Value

- [ ] Change the `aria-valuetext` back to just using numbers (SO SORRY!) I hope this is not a big problem.
- Initial focus responses: 
    - 1, Left Value, number spinner
    - 1, Left Value, number spinner
- on changing
    - 2, 3, 4, etc

- [ ] Change the help text for the number spinners to
- Set left and right values to create your challenge, then move hands to explore it.

- [ ] Create a new pattern for the Context Response that uses what we had in the `aria-valuetext`:
    - Current challenge {{left-value}} to {{right-value}}. Hands, {{proximity-to-ratio}} challenge ratio. 
        - EXAMPLE: Current challenge 2 to 2. Hands, somewhat close to challenge ratio.

**Comment by terracoda** [comment]
And, BTW, nice work on previous implementations. Hopefully, these changes will make it even better.

**Comment by zepumph** [comment]
Even if we get rid of it, that comma was easy

**Comment by zepumph** [comment]
> (SO SORRY!) I hope this is not a big problem.

Not a worry! It was simple. 


> Current challenge 2 to 2. Hands, somewhat close to challenge ratio.

What do you think about adding "new" like we did over in https://github.com/phetsims/ratio-and-proportion/issues/196

**Comment by terracoda** [comment]
@zepumph, good catch, I think using "new" will work perfectly, here, too.

However, just an FYI, the response upon opening the My Challenge accordion (#211), won't sound right with "new" because not change has been made.

**Comment by terracoda** [comment]
Oh, forget that last comment, #211 is just using the first part of the above response:
- "Current challenge 2 to 2."

**Comment by terracoda** [comment]
@zepumph, all we need is "new" to align with what we did in #196.
- labels for spinners are good.
- help text is good.
- As you suggested, just add "new" to the new Context Response pattern, and I think this issue can be closed.

**Comment by terracoda** [comment]
Once #211 is implemented, I think the My Challenge accordion box and number spinners should be complete.

**Comment by terracoda** [comment]
Well, there could be minor tweak in #212, and then I need to re-verify how things sound before we do anything with #213. It already sounds better.

**Comment by zepumph** [comment]
@terracoda, "new" was added above. See you next over in https://github.com/phetsims/ratio-and-proportion/issues/211

**Comment by terracoda** [comment]
The "new" sounds great. Closing this issue.
